web/src/app/models/userSocial.model.ts

```diff
@@ -1,10 +1,20 @@
 export interface UserSocial {
-  github: {
+  github?: {
     updatedAt: string,
     followers: number,
     following: number,
   };
-  twitter: {
+  twitter?: {
+    updatedAt: string,
```

Comment:
Might be worth creating a `social` model?

```typescript
{
     updatedAt: string;
     followers: number;
     following: number;
}
```

Reply:
yes @eddiejaoude . Will do it.